docs/User/ProjectRelated/Changelog.md

```diff
@@ -3,6 +3,19 @@
 <!-- See rendered changelog at https://streamly.composewell.com -->
 ## Unreleased
 
+* The unboxed arrays now use values of Prim over Storable. Please note/follow
+  the steps below for migration.
+  1. Make sure the values that you store in an unboxed array have the `Prim`
+     instance.
+  2. Replace the following import,
+     ```
+     import Foreign.Storable (Storable(alignment, peek, poke, sizeOf))
+     ```
+     with,
+     ```
+     import Streamly.Internal.Data.Unboxed (Storable, alignment, peek, poke, sizeOf)
+     ```
```

Comment:
Is this even required? This can be phrased in a better way.
Use the constraint `Prim` instead of `Storable`.

Reply:
This is internal, nobody should even be aware of peek/poke/sizeOf from this module. In fact, this entire module is internal. Users should just use Prim.

Reply:
Yeah, I've changed it.